doc/man3/BIO_ADDRINFO.pod

```diff
@@ -92,10 +92,7 @@ information they should return isn't available.
 =head1 NOTES
 
 The BIO_lookup_ex() implementation uses the platform provided getaddrinfo()
```

Comment:
I would either merge this implementation detail into the main description of `BIO_lookup_ex` explaining the naming of the manpage or else drop the `NOTES` section. Also, shouldn't there be a hyphen in "platform-provided"?

Reply:
yes to both.  I hate the OpenSSL cop-out of writing "Notes" sections.